### PR TITLE
Add Ruby 3 support. Update dependencies, theme to MonokaiSublime.

### PR DESCRIPTION
**Pull request by rohithmadhavan**
Our TravisCI builds were using Ruby `2.3.3` and `2.4.0` to build, both of which are now EOL. This diff uses changes from upstream to update dependencies and add Ruby `3` support, which is crucial to build and test locally. 

Pinning the node version used by TravisCI to the latest LTS release: 16.14.2, since the latest node version 18.0.0 causes a [build failure](https://app.travis-ci.com/github/SnapLogic/developer.snaplogic.com/builds/249712001), also [reported in TravisCI Community](https://travis-ci.community/t/the-command-npm-config-set-spin-false-failed-and-exited-with-1-during/12909).

Updated the theme to MonokaiSublime, it has better syntax highlighting.

Before:
Screen Shot 2022-04-21 at 1 01 51 PM

After:
Screen Shot 2022-04-21 at 1 02 02 PM

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -8,12 +8,14 @@ Vagrant.configure(2) do |config|
   config.vm.provision "bootstrap",
     type: "shell",
     inline: <<-SHELL
-      sudo apt-add-repository ppa:brightbox/ruby-ng
+      # add nodejs v12 repository
+      curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+
       sudo apt-get update
-      sudo apt-get install -yq ruby2.4 ruby2.4-dev
+      sudo apt-get install -yq ruby ruby-dev
```
Comment (ddellsperger) [inline Vagrantfile:15]: I think we're fine with this, but this specifically installs ruby 3, right?

Comment (rohithmadhavan) [reply Vagrantfile:15]: Whatever `ruby` and `ruby-dev` point to in Ubuntu's repositories. For `bionic` that's `2.5.1`, but newer releases use `3`.

**Review by ddellsperger — APPROVED** [review]
Looks good to me, should we also update the readme with new requirements for running locally? or does 2.3.1 still technically work with all of the updates to the Gemfile?

**Comment by rohithmadhavan** [comment]
> should we also update the readme with new requirements for running locally

Yes, we now need Ruby 2.5+, updated README.

**Pull request merged**